### PR TITLE
126 update colab notebook

### LINKED ISSUE #126: Update Colab Example Notebook
Update the Armory Colab example notebook for library updates.

### PR DESCRIPTION
**Pull request by treubig26**
Closes #126

### CODE REVIEW

File: examples/notebooks/object_detection_license_plates.ipynb [hunk 1]
```diff
@@ -29,6 +29,20 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "batch_size = 2\n",
```
Comment (deprit) [inline examples/notebooks/object_detection_license_plates.ipynb:42]: These constants should be explained in a comment cell.

**Review by deprit — COMMENTED** [review]
Walkthrough notebook is much improved -- it now provides detailed introduction to library.

File: examples/notebooks/armory_evaluation_walkthrough.ipynb [hunk 2]
```diff
@@ -0,0 +1,1121 @@
...
+    },
+    "id": "eFB-V0Y1RYJ3",
+    "outputId": "ee82d90c-bf14-4f46-866a-41f2b80e228b"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: armory-library in /home/kyle-treubig/Code/jatic/armory/.venv-3.11/lib/python3.11/site-packages (24.1.post72+g08fb62fc.d20240207)\n",
```
[the remark this answers is not in the document]
Comment (treubig26) [reply examples/notebooks/armory_evaluation_walkthrough.ipynb:47]: I added this tag, but it didn't seem to have any effect. I can configure it to be removed by the nbstripout pre-commit hook.

File: examples/notebooks/armory_evaluation_walkthrough.ipynb [hunk 3]
```diff
@@ -0,0 +1,1121 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataloader = ImageClassificationDataLoader(\n",
+    "    hf_dataset,\n",
+    "    dim=ImageDimensions.CHW,\n",
+    "    scale=Scale(\n",
+    "        dtype=DataType.FLOAT,\n",
+    "        max=1.0,\n",
```
[the remark this answers is not in the document]
Comment (treubig26) [reply examples/notebooks/armory_evaluation_walkthrough.ipynb:548]: While that happens to be the same value in this case, it isn't always. The max defined here is the unnormalized max, not the normalized max. If the normalization used ImageNet mean & std dev, `image_max` would have a value close to `3.0`, but max here should still be `1.0`.

In general, a `dtype=DataType.FLOAT` always has a `max=1.0` while a `dtype=DataType.UINT8` would have a `max=255`.

File: examples/notebooks/armory_evaluation_walkthrough.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (deprit) [reply examples/notebooks/armory_evaluation_walkthrough.ipynb:47]: `hide-output` does not work in Colaboratory. I was able to hide the output using the UI, and it stays hidden when the notebook is uploaded. The UI adds the `collapsed` tag below.
```
        {
            "cell_type": "code",
            "execution_count": 1,
            "metadata":
            {
                "colab":
                {
                    "base_uri": "https://localhost:8080/"
                },
                "id": "eFB-V0Y1RYJ3",
                "outputId": "424e8232-9bb7-46ed-eeaf-0a13b05a6969",
                "collapsed": true,
                "executionInfo":
                {
                    "status": "ok",
                    "timestamp": 1709742568522,
                    "user_tz": 300,
                    "elapsed": 60088,
                    "user":
                    {
                        "displayName": "Etienne Deprit",
                        "userId": "00561617842413560974"
                    }
                }
            },
            "outputs":
```
Here's what it looks like in Colab.
image

**Review by deprit — APPROVED** [review]
The walkthrough notebook runs in Cola -- looks good!
